src/exampleTypes/TermsStatsTable.js

```diff
@@ -61,7 +61,7 @@ let TermsStatsTable = ({
   ...props
 }) => (
   <div>
-    <Flex style={{ ...toolBarStyle, margin: 40, marginBottom: 0 }}>
+    <Flex style={{ ...toolBarStyle, margin: '0 8px' }}>
```

Comment:
this feels out of place given that it's a grey vest opinionated spacing 🤔 

good catch killing `margin:40` thought 👍